mongo.js

```diff
@@ -0,0 +1,37 @@
+'use strict'
+const config = require('config')
+const MongoClient = require('mongodb')
+
+module.exports = function (koop) {
+
+  this.getData = (req, callback) => {
+    var url = config.get('mongo.connectionUrl')
+    MongoClient.connect(url, (err, db) => {
+      var features = []
+      var collection = db.collection('restaurants')
+      var stream = collection.find({}).stream()
+      stream.on('data', (doc) => {
+        features.push(this.formatFeature(doc))
```

Comment:
makes sense to translate the features as they come in, rather than waiting for the full batch